src/App.js

```diff
@@ -61,7 +62,7 @@ export default function App() {
         <Route
           path="/profile/:spookmarked"
```

Comment:
Why do you use a parameter here? It seems that you only want to use the sub route `profile/spookmarked`. You only need to use the `:` colon if you want a dynamic route, i.e. if the route can have different working strings like `profile/page1`, `profile/someotherRoute` and `profile/aThirdDynamicRoute` All the prior routes would be catched by the dynamic route `profile/:spookmarked` and the `name` of the route would be saved in the `params` object, e.g. `{spookmarked: 'page1'}` or `{spookmarked: 'someotherRoute'}.